include/revng/Pipes/PipelineManager.h

```diff
@@ -88,12 +88,12 @@ class PipelineManager {
 
   /// Triggers the full serialization of every step and every container to the
   /// the specified DirPath or the Execution directory if omitted.
-  llvm::Error storeToDisk(llvm::StringRef DirPath = llvm::StringRef());
+  llvm::Error storeToDisk(revng::Path *DirPath = nullptr);
 
   /// Trigger the serialization of a single Step to the specified DirPath or
   /// the Execution directory if omitted.
   llvm::Error storeStepToDisk(llvm::StringRef StepName,
-                              llvm::StringRef DirPath = llvm::StringRef());
+                              revng::Path *DirPath = nullptr);
```

Comment:
Does anyone actually use this argument? If not, drop it.

Reply:
It's used by `PipelineC.cpp` in `rp_manager_save`. This was added to support the `FIFO` synchronization. I'd drop it, but I'm double checking if you see some other use for it.

Reply:
Dropped it and the ancilliary functions in PipelineC